package.json

```diff
@@ -57,6 +58,7 @@
     "@truffle/hdwallet-provider": "1.0.34",
     "@types/chai": "^4.2.12",
     "@types/chai-as-promised": "^7.1.3",
+    "@types/cli-table": "^0.3.0",
```

Comment:
duplicate: you also add it as a devDependency
(and anyway ,need to remove many deps since they are devDeps)

Reply:
not relevant after 'workspaces' PR